### ISSUE TITLE
Not allowed to load local resource: blob:null/88ca3529-3169-498e-a4f0-95b53f569642

### ISSUE THREAD

**Issue by xclss99** [finos/perspective#1720]
I created a CDN demo according to the JS guide on the official document. When opening HTML, the console reported this error: not allowed to load local resource: blob: null / 88ca3529-3169-498e-a4f0-95b53f569642.
I copied all the code of the streaming example from the example on GIT and reported a similar error when opening HTML: not allowed to load local resource: blob: null / 88ca3529-3169-498e-a4f0-95b53f569642. What is the reason?

**Comment by texodus** [comment]
Without a repro, all I can offer is to type your question into Google on your behalf and paste the first result:

[(StackOverflow) Getting "Not allowed to load local resource" error while trying to attach a MediaSource object as the source of a HTML5 video tag](https://stackoverflow.com/questions/17682978/getting-not-allowed-to-load-local-resource-error-while-trying-to-attach-a-medi)

TL;DR you need to host the file, not use a `file://` protocol.
